### PR TITLE
Fix broken CI build

### PR DESCRIPTION
**Pull request by shilman**
Issue: N/A

## What I did

- restore code from
https://github.com/storybooks/storybook/commit/cba07adda085fb3a8d7850f2b
53673be95196c3e
- update test fixture for codemod

## How to test

```
yarn test --runInBand --core --reactnative
```

Is this testable with jest or storyshots?

Does this need a new example in the kitchen sink apps?

Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: lib/codemod/src/transforms/__testfixtures__/update-organisation-name/update-organisation-name.input.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /* eslint-disable */
 import '@kadira/storybook-addons';
 import '@kadira/react-storybook-decorator-centered';
-import '@kadira/storybook-addon-comments';
```
Comment (Hypnosphi) [inline lib/codemod/src/transforms/__testfixtures__/update-organisation-name/update-organisation-name.input.js]: What if a project actually has this dependency? Currently it's replaced with `@storybook/react-addon-comments`, which is a non-existent package

**Review by Hypnosphi — CHANGES_REQUESTED** [review]

File: lib/codemod/src/transforms/__testfixtures__/update-organisation-name/update-organisation-name.input.js (hunk 1 above)
Comment (Hypnosphi) [reply lib/codemod/src/transforms/__testfixtures__/update-organisation-name/update-organisation-name.input.js]: See also: https://github.com/storybooks/storybook/commit/cf054ad1b35a7e934c18e10db411bff8d39aaa8d#commitcomment-25450913

Comment (shilman) [reply lib/codemod/src/transforms/__testfixtures__/update-organisation-name/update-organisation-name.input.js]: Good point 😄

Comment (shilman) [reply lib/codemod/src/transforms/__testfixtures__/update-organisation-name/update-organisation-name.input.js]: @Hypnosphi @danielduan any idea how this passed in https://github.com/storybooks/storybook/pull/2189 

I'm not so familiar with how our codemod works?

Comment (Hypnosphi) [reply lib/codemod/src/transforms/__testfixtures__/update-organisation-name/update-organisation-name.input.js]: #2189 doesn't change anything in codemod

Comment (shilman) [reply lib/codemod/src/transforms/__testfixtures__/update-organisation-name/update-organisation-name.input.js]: Updated codemod & test cases per your suggestions @Hypnosphi . Also ran `npm deprecate`

**Review by Hypnosphi — APPROVED** [review]

**Pull request merged**